src/librssguard/gui/feedsview.cpp

```diff
@@ -231,6 +231,10 @@ void FeedsView::editSelectedItem() {
                           QSystemTrayIcon::MessageIcon::Warning});
   }
 
+  RootItem* selected_item = selectedItem();
```

Comment:
why?

Reply:
When the feed's settings dialog closes, the messages panel doesn't change the layout direction.
So, I emit a signal to refresh the layout direction of the selected feed.
If there is anther way to do that, I don't know.

```
RootItem* selected_item = selectedItem();
emit itemSelected(selected_item);
```